test/app.test.js

```diff
@@ -871,14 +874,14 @@ describe('app', function() {
       require('../lib/builtin-models')(app.registry);
       var db = app.dataSource('db', {connector: 'memory'});
 
-      app.enableAuth({dataSource: 'db'});
-
-      expect(Object.keys(app.models)).to.include.members(AUTH_MODELS);
+      return app.enableAuth({dataSource: 'db'}).then(() => {
```

Comment:
This change shouldn't be needed, because we are attaching the models synchronously in the current implementation. Unless I am missing something?